### ISSUE TITLE
Don't display bookmark of empty title

### ISSUE THREAD

**Issue by tuchida** [CodeRect/bookmark-tab#2]
This addon is simple and very cool!!!

I found difference from Firefox's bookmark.
If registered empty title, Firefox display the URL. But this is not the case.
![screenshot_from_2016-01-09 18 15 48](https://cloud.githubusercontent.com/assets/201790/12215397/54da3cca-b6fe-11e5-8a59-06e952ea85ba.png)

For example:
https://github.com/CodeRect/bookmark-tab/blob/master/index.js#L198
```diff
-								'strTitle': PlacesUtils.bookmarks.getItemTitle(intIdent),
+								'strTitle': PlacesUtils.bookmarks.getItemTitle(intIdent) || PlacesUtils.bookmarks.getBookmarkURI(intIdent).spec,
```
(Sorry. I have not tried this. I could not build the addon on Linux.)

**Comment by CodeRect** [comment]
hi tuchida,

thank you very much for this suggestion - i went ahead and implemented it in commit bf538ac

i created a new build for you, in case you would like to use the updated version ahead of time. in order to do so you however need to use at least firefox 44 and disable xpinstall.signatures.require in the about:config page: https://github.com/CodeRect/data-storage/blob/master/bookrect.xpi

thanks again - simon

**Comment by tuchida** [comment]
Thanks!
It work as expected.
